core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/quiverPlot/QuiverPlotOpDesc.scala

```diff
@@ -0,0 +1,116 @@
+package edu.uci.ics.texera.workflow.operators.visualization.quiverPlot
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+@JsonSchemaInject(json = """
+{
+  "attributeTypeRules": {
+    "value": {
+      "enum": ["integer", "long", "double"]
+    }
+  }
+}
+""")
+class QuiverPlotOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+
+  //property panel variable: 4 requires: {x,y,u,v}, 2 optional: {x,y} for specific points
+
+  @JsonProperty(value = "x", required = true)
+  @JsonSchemaTitle("x")
+  @JsonPropertyDescription("column for the x-coordinate of the starting point")
+  @AutofillAttributeName var x: String = ""
```

Comment:
The property check doesn't work for `x`, `y`, `u`, `v`. When I use a column of type `string`, there is no warning and the graph cannot be plotted. Please fix the type check.